serverless/src/functions/apps/handler.ts

```diff
@@ -0,0 +1,107 @@
+import { APIGatewayProxyResult, APIGatewayEvent } from 'aws-lambda';
+import { formatJSONResponse } from '@libs/api-gateway';
+import * as dotenv from 'dotenv';
+import { v4 } from 'uuid';
+import { prisma } from '@libs/prisma';
+import {
+  BunnyCdn,
+  BunnyCdnError,
+  CreatePullZoneMethodArgs,
+} from '@libs/bunnyCDN';
+
+export const submitAppInfo = async (
+  event: APIGatewayEvent
+): Promise<APIGatewayProxyResult> => {
+  try {
+    // Check the parameters and environment variables
+    dotenv.config();
+    if (event.body === null || process.env.BUNNY_CDN_ACCESS_KEY == undefined) {
+      return formatJSONResponse({
+        status: 422,
+        message: 'Required parameters were not passed.',
+      });
+    }
+
```

Comment:
Should we check if the app existing in the DB before we create a new one? Or will the create at the bottom of the file do an upsert?

Reply:
Line 78 should check that.